src/routes/docs/products/auth/oauth2/+page.markdoc

```diff
@@ -36,7 +36,7 @@ const client = new Client()
 const account = new Account(client);
 
 // Go to OAuth provider login page
-account.createOAuth2Session('github', '[LINK_ON_SUCCESS]', '[LINK_ON_FAILURE]', true);
+account.createOAuth2Session(OAuthProvider.GitHub, '[LINK_ON_SUCCESS]', '[LINK_ON_FAILURE]');
```

Comment:
It's github instead of GitHub since GitHub will be taken as incorrect from what I understand

Reply:
Nope for node is it PascalCase